Tests/DataXScenarios/DataXScenarios/SaveAndDeploy.cs

```diff
@@ -69,10 +87,10 @@ public static StepResult GenerateConfigs(ScenarioContext context)
             string jsonResult = Request.Post(baseAddress,
                     new RequestContent(Encoding.UTF8.GetBytes(json), "application/json"),
                     context[Context.AuthToken] as string);
-            
+
             dynamic result = JObject.Parse(jsonResult);
             context[Context.GenerateConfigsRuntimeConfigFolder] = (string)result.result.Properties.runtimeConfigFolder;
-            
+
             string generateConfigsRuntimeConfigFolder = context[Context.GenerateConfigsRuntimeConfigFolder] as string;

```

Comment:
why you crate generateConfigsRuntimeConfigFolder? 

Can we directly use context[Context.GenerateConfigsRuntimeConfigFolder] like this?

return  new StepResult(!string.IsNullOrWhiteSpace(return new StepResult(!string.IsNullOrWhiteSpace(context[Context.GenerateConfigsRuntimeConfigFolder]),

Reply:
To remove redundancy. This variable is being used multiple times